### PR TITLE
adding SECRET_KEY to staging

### PR DESCRIPTION
**Pull request by kammerdiener**

### CODE REVIEW

File: infra/staging/index.ts [hunk 1]
```diff
@@ -21,6 +21,8 @@ let baseUrl = `http://${domain}/`;
 
 let sentryDSN = `${process.env["SENTRY_DSN"]}`;
 
+let secretKey = `${process.env["SECRET_KEY"]}`
```
Comment (aminah-io) [inline infra/staging/index.ts:24]:
```suggestion
let secretKey = `${process.env["SECRET_KEY"]}`;
```

Comment (aminah-io) [reply infra/staging/index.ts:24]: Just a missing semicolon, otherwise LGTM!

**Review by aminah-io — APPROVED** [review]
LGTM 🚀

**Review by nutrina — APPROVED** [review]
LGTM

**Pull request merged**